README.md

```diff
@@ -65,6 +65,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Danfe Solution Pvt. Ltd.](http://danfesolution.com/) | Gyaneshwor, Kathmandu | IT Company focusing on Web Development, Mobile Application and Desktop Application
 - [Danphe Software Labs Pvt. Ltd.](http://www.danpheit.com/) | Anamnagar, Kathmandu  | We specialize in Software and Web Development
 - [Deerwalk](http://www.deerwalk.com/) | Sifal, Kathmandu | Bringing you software solutions for POPULATION HEALTH MANAGEMENT
+- [Dolphin Dive Technology Pvt. Ltd.](https://dolphindivetech.com/) | Jhamsikhel, Lalitpur |  US Based Outsourcing Agency
```

Comment:
@pradip7374 Need a good description for the company here. `US Based Outsourcing Agency` doesn't say much. Also, let's remove the region (US) from the description and focus on what the company does best.